python/packages/streamsx/rest_primitives.py

```diff
@@ -49,6 +50,21 @@ def __init__(self, json_rep, rest_client):
 
     def __str__(self):
         return pformat(self.__dict__)
+    __repr__ = __str__
+
+    def refresh(self):
+        """Make a REST call to refresh the JSON object stored within the object
+
+        Returns:
+            _ResourceElement: The provided object instance is returned with updated value
+        """
+        rest_client = self.rest_client
+        rest_self = self.rest_self
+        json_rep = rest_client.make_request(rest_self)
+        self.__dict__.clear()
```

Comment:
I think it would be better to have the `__init__` create a method that sets the fields from the REST response and then call that method on a refresh, rather than `__init__`.